packages/attestation-service/migrations/20200506234935-attestation-v2.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const transaction = await queryInterface.sequelize.transaction()
+
+    try {
+      await queryInterface.removeColumn('Attestations', 'phoneNumber')
+
+      await queryInterface.addColumn('Attestations', 'identifier', {
+        type: Sequelize.STRING,
+      })
+
+      await queryInterface.removeIndex(
+        'Attestations',
+        ['account', 'phoneNumber', 'issuer'],
+        { fields: ['account', 'phoneNumber', 'issuer'], unique: true },
+        { transaction }
+      )
+
+      await queryInterface.addIndex(
+        'Attestations',
+        ['account', 'identifier', 'issuer'],
+        { fields: ['account', 'identifier', 'issuer'], unique: true },
+        { transaction }
+      )
+
+      await queryInterface.addIndex(
+        'Attestations',
+        ['createdAt'],
+        { fields: ['createdAt'], unique: true },
```

Comment:
I'm not sure how common it is to add a uniqueness constraint on a timestamp. We could have 2 records at the same time, right?

Reply:
good point -- will update